css/styles.css

```diff
@@ -1,52 +1,57 @@
-* {
+*, *::before, *::after {
 	box-sizing: border-box;
 }
 
+:root {
+	font-size: 15px;
+}
+
 body {
-	background-color: #fff;
+	background: #fff;
+	color: #000;
 }
 
 a {
 	text-decoration: none;
 	color: rgba(71, 71, 107, .7);
 }
 
+a:hover, a:focus {
+	color: #000;
+}
+
 .container {
 	max-width: 1200px;
+	padding: 0 20px;
 	margin: 0 auto;
 	text-align: center;
 }
 
 /* header and navigation menu */
 
-.minimo {
+.logo {
 	padding-top: 30px;
 	padding-bottom: 20px;
 }
 
-.main-nav ul {
-	margin: 0 auto;
-}
 
 .main-nav li {
-	font-family: "Playfair Display", serif;
+	padding: 5px 0;
 	color: rgba(71, 71, 107, .7);
-	font-size: 0.8rem;
-	font-weight: 500;
+	font: 0.8rem "Playfair Display", serif;
 	text-transform: uppercase;
-	padding: 5px 0;
+}
+
+.main-nav li:last-child {
+	padding-right: 0;
 }
 
 .header-photo {
-	background-image: url("../images/Header_photo.jpg");
-	background-repeat: no-repeat;
-	background-position: center;
-	height: 600px;
+	margin-bottom: 60px;
 }
 
 .header-photo img {
-	margin: 0 auto;
-	width: 90%;
+	max-width: 100%;

```

Comment:
Też sprawdziłem i `1200px` to szerokość canvasa czyli całego obszaru roboczego. Sam kontent strony ma `1060px` szerokości. Biorąc pod uwagę, że contener ma `40px` padding z każdej strony (powyżej 768px) to szerokość dla `.container` to powinno być `1140px` (1060 + 40 + 40). Jako, że masz ustwiony `box-sizing: border-box;` to paddingi i bordery są "zabierane" z szerokości kontenera.

![szerokosc](https://cloud.githubusercontent.com/assets/7902324/19364285/d2c1143c-918d-11e6-9985-6b948a889195.png)